ports/llgi/portfile.cmake

```
@@ -0,0 +1,60 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
+# see https://github.com/altseed/LLGI/pull/150
+vcpkg_download_distfile(LLGI_PR_150_PATCH
+    URLS "https://patch-diff.githubusercontent.com/raw/altseed/LLGI/pull/150.diff"

```

```suggestion
    URLS "https://patch-diff.githubusercontent.com/raw/altseed/LLGI/pull/150.diff?full_index=1"
```
This modification is to avoid the common changes in the abbreviated sha length in dynamic patch index lines affecting the download of the patch. See PR #33309 and https://github.com/microsoft/vcpkg/issues/33305#issuecomment-1687510280.